services/ProductRegistrationService.php

```diff
@@ -88,6 +88,8 @@ private function optInStrategy($email) {
         curl_setopt($curl, CURLOPT_POST, true);
         curl_setopt($curl, CURLOPT_POSTFIELDS, http_build_query(array('email' => $email)));
         curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
+        curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, false);
+        curl_setopt($curl, CURLOPT_SSL_VERIFYHOST, 2);
```

Comment:
This is just declaring what should be default, but agree it makes sense since not forcing verifypeer.